lib/administrate/page/form.rb

```diff
@@ -10,8 +10,13 @@ def initialize(dashboard, resource)
 
       attr_reader :resource
 
-      def attributes
-        dashboard.form_attributes.map do |attribute|
+      def attributes(action = nil)
+        action = case action
+                 when "update" then "edit"
+                 when "create" then "new"
+                 else action
+                 end
```

Comment:
I have a very personal yet deep aversion to this sort of indentation :see_no_evil: How about the following?

```suggestion
        action =
          case action
          when "update" then "edit"
          when "create" then "new"
          else action
          end

```

Also added a space at the end.